publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -484,7 +489,7 @@ public void shutdownCompleted(ShutdownSignalException cause) {
         } catch (Exception e) {
             log.error(e.getMessage(), e);
             if(!channel.isOpen()&& rabbitConnection.isOpen()){
-                throw new RemRemPublishException("Channel was closed for Rabbitmq connection ::" + factory.getHost() + factory.getPort());
+                throw new RemRemPublishException("Channel was closed for Rabbitmq connection ::" + factory.getHost() + factory.getPort()+" due to "+e.getMessage());
```

Comment:
Rabbitmq -> RabbitMq for consistency

Rabbitmq connection ::" + factory.getHost() + factory.getPort()+" due to "+e.getMessage());
Won't this squash the Host + Port without separator making it hard to read?
A ':' b/w host:port is commonly used I believe.

Reply:
Changed it